build.cake

```diff
@@ -20,7 +22,17 @@ var OWNERS = new [] { "Charlie Poole" };
 var DESCRIPTION = "This extension allows NUnit to recognize and load NUnit projects, file type .nunit.";
 var SUMMARY = "NUnit Engine extension for loading NUnit projects.";
 var COPYRIGHT = "Copyright (c) 2016 Charlie Poole";
-var RELEASE_NOTES = new [] { "See https://raw.githubusercontent.com/nunit/nunit-project-loader/master/CHANGES.txt" };
+var RELEASE_NOTES = new [] { 
```

Comment:
Actually, it was an invalid link, since we never had the file before! But I see your point about adding an extra step, which we could easily avoid. I'll put the link back.

For now, I prefer to maintain the file as text, since it is distributed in many more ways than via GitHub. We should come back to that when we start automating the releases.